README.md

```diff
@@ -100,137 +66,85 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Use of PostgreSQL for databases and tables.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Coming soon](https://github.com/Nkaleth/Vet_clinic_database)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- visual studio
+- github
+- PosgreSQL 15
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
+  mkdir my-folder
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com:Nkaleth/Vet_clinic_database.git
 ```
---->
 
-### Install
+ To access the cloned directory, run:
 
-Install this project with:
-
-<!--
-Example command:
+ ```sh
+ cd Vet_clinic_database
+ ```
 
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
+### Install
 
-<!--
-Example command:
+Install this project with:
 
-```sh
-  rails server
 ```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
+  npm install
 ```
```

Comment:
- [ **OPTIONAL** ] To improve your documentation be more professional, I think it would be better if you remove the above command as you don't need to install dependencies for this project. You can leave there an other information instead such as `N/A`, `No need to install the app`or some thing similar ti this,... 
Good job :100: :100: